pom.xml

```diff
@@ -184,7 +190,6 @@
       <groupId>io.vertx</groupId>
       <artifactId>vertx-core</artifactId>
       <version>${vertx.version}</version>
-      <scope>test</scope>
```

Comment:
It should be test scope in case removing `folio-service-tools` dependency

Reply:
fixed